src/main/java/me/karakelley/tictactoe/IO.java

```diff
@@ -20,5 +21,25 @@ public void display(String message) {
   public String getInput() {
     return inputScanner.nextLine();
   }
+
+  public int anyKey() throws IOException, InterruptedException {
+    int input = -1;
+    enterCharacterMode();
+    while (input < 0 ) {
+      input = in.read();
+    }
+    enterLineMode();
+    return input;
+  }
+
+  private void enterCharacterMode() throws IOException, InterruptedException {
+    String[] cmd = new String[]{"/bin/sh", "-c", "stty raw </dev/tty"};
```

Comment:
I don't love this because I think it makes your code platform-dependent (It won't run on windows anymore). I don't think there's a way around it, though.